pom.xml

```
@@ -384,30 +384,28 @@
 
 
         <!-- ###  PAYARA  ### -->
-
         <profile>
-            <id>payara4</id>
+            <id>Payara Community</id>
             <activation>
                 <property>
                     <name>payara.version</name>
-                    <value>/4.*/</value>
+                    <value>/^[5-9]\.[0-9]{4}(-SNAPSHOT|\..*)*/</value>
                 </property>
             </activation>
             <properties>
-                <payara.arquillian.version>1.4</payara.arquillian.version>
-                <payara.directory.name>payara41</payara.directory.name>
-                <payara.version.major>4</payara.version.major>
-                <payara_domain>payaradomain</payara_domain>
-                <skipServletClientCertificateCommonName>true</skipServletClientCertificateCommonName>
+                <payara.directory.name>payara5</payara.directory.name>
+                <payara.version.major>5</payara.version.major>
+                <payara_domain>production</payara_domain>
+                <payara.arquillian.version>2.2</payara.arquillian.version>
             </properties>
         </profile>
 
         <profile>
-            <id>payara5</id>
+            <id>Payara Enterprise</id>
             <activation>
                 <property>
                     <name>payara.version</name>
-                    <value>/5.*/</value>
+                    <value>/^[5-9]\.[0-9]{2}\.[0-9]{1,2}(-SNAPSHOT)*/</value>
```

Same argument as above, I don't see the need to explicitly say it ends in snapshot.

```suggestion
                    <value>/^[5-9]\.[0-9]{2}\.[0-9].*/</value>
```

---

Mindset was that a version will always be the format X.XX.X(X)(-SNAPSHOT) and so to only allow valid version numbers that should be what the regex checks but am happy to make the changes to both